build.sbt

```diff
@@ -35,7 +35,21 @@ lazy val commonSettings: Seq[sbt.Def.Setting[_]] = Seq(
 )
 
 lazy val commonJsSettings: Seq[sbt.Def.Setting[_]] = Seq(
-  Test / scalaJSLinkerConfig ~= { _.withModuleKind(ModuleKind.CommonJSModule) }
+  Test / scalaJSLinkerConfig ~= { _.withModuleKind(ModuleKind.CommonJSModule) },
+  scalacOptions ++= {
+    // Map the sourcemaps to github paths instead of local directories
+    val flag =
+      if (scalaVersion.value.startsWith("3")) "-scalajs-mapSourceURI"
```

Comment:
Copy+Pasta: Tyrian isn't released for Scala 2. :smile:

Reply:
Another reason why I should never copy paste without due diligence :P